internal/packmgr/installer.go

```
@@ -0,0 +1,247 @@
+		return fmt.Errorf("缺少依赖: %s", strings.Join(missing, "; "))
+	}
+	return nil
+}
+
+// copyFile 复制文件并保留源权限位。
+func copyFile(src, dst string) error {
+	source, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer source.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `source.Close` is not checked

(errcheck)